colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,984 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb_registry/zoo_wandb.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+    "<!--- @wandbcode{zoo-wandb} -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d75eb97c",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://wandb.me/logo-im-png\" width=\"400\" alt=\"Weights & Biases\" />\n",
+    "\n",
+    "<!--- @wandbcode{zoo-wandb} -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f7e5952",
+   "metadata": {},
+   "source": [
+    "# Weights & Biases Registry Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "507ab87f",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to demonstrate how you and other members of your organization can use W&B Registry to track, share, and use dataset and model artifacts in your machine learning workflows. By the end of this notebook, you will know how to use W&B to:\n",
+    "\n",
+    "1. Create [collections](https://docs.wandb.ai/guides/registry/create_collection) within the [Dataset and Model registry](https://docs.wandb.ai/guides/registry/registry_types#core-registry).\n",
```

Comment:
This should just be Registry no? We are trying to make it abstract to more than just models and artifacts?